go.mod

```diff
@@ -1,103 +1,120 @@
 module github.com/xpladev/xpla
 
-go 1.21
+go 1.23
```

Comment:
for use toolchain, I think using go 1.23.2 is better

Reply:
https://github.com/xpladev/xpla/pull/154/commits/dfc5138909f06fcab10b2cc3a4e240a48873853a